decidim-proposals/app/views/decidim/proposals/admin/proposals/index.html.erb

```diff
@@ -58,7 +58,7 @@
           </th>
 
           <th>
-            <%= sort_link(query, :state, [:state, :is_emendation], t("models.proposal.fields.state", scope: "decidim.proposals") ) %>
+            <%= t("models.proposal.fields.state", scope: "decidim.proposals") %>
```

Comment:
I'm removing sorting by state because it doesn't work and I didn't find an easy fix.